test/e2e/framework/job/wait.go

```diff
@@ -65,10 +65,19 @@ func WaitForJobFinish(c clientset.Interface, ns, jobName string) error {
 		if err != nil {
 			return false, err
 		}
-		return jobutil.IsJobFinished(curr), nil
+		return isJobFinished(curr), nil
 	})
 }
 
+func isJobFinished(j *batchv1.Job) bool {
```

Comment:
How to deal with the [original function](  https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/job/utils.go#L24), is it copying this function now?